File: PATTERNS/Pattern05/Pattern5.py
    # * * * * *
    #  * * * *
    #   * * *
    #    * *
     #    *

n = int(input("Enter the row number:"))

for i in range(1,n+1):
     for k in range(1,i+1):
        print(" ",end='')
     for j in range(i,n+1):
         print("* ",end='')
     print()

File: PATTERNS/Pattern05/Pattern5.go
// * * * * *
//  * * * *
//   * * *
//    * *
//     *

package main

import "fmt"

func main() {

	var n int;
	fmt.Printf("Enter the row number:")
	fmt.Scanln(&n);

	for i := 1; i <= n; i++ {
		for k := 1; k <= i; k++ {
			fmt.Printf(" ")
		}
		for j := i; j <= n; j++ {
			fmt.Printf("* ")
		}
		fmt.Printf("\n")
	}
}
